app/models/shipit/deploy.rb

```diff
@@ -21,7 +21,9 @@ class Deploy < Task
         'pending' => 'pending',
         'running' => 'in_progress',
         'failed' => 'failure',
+        'timedout' => 'failure',
         'success' => 'success',
+        'faulty' => 'error',
```

Comment:
I'm not sure I understand how and when we differentiate error vs. failure.

IMO something involving user action or general deploy state being 'bad' is how I view error, and the deploy operation failing to finish is what I call 'failure'.

Reply:
Yeah that breakdown makes sense. Specifically error would be things that are not retriable, and failure are things that are imo.

Reply:
I tend do think of them the other way around to you Jack 😅 Let's not stress over the semantics for now, it's more important to plug the gap in statuses.